### ISSUE TITLE
Allow a user to list out all the examples for a group

### ISSUE THREAD

**Issue by jelaplan** [zowe/zowe-cli#409]
**Goal**: Allow a user to list out all the examples for a group. This is a similar feature request as with #408. Examples are the most popular area of the help pages so a concise merged listing would be an easy way to find, discover, and quickly use commands. 

**Overarching problem**: Improving the Findability and Discoverability of Capabilities and Functionality. We hear customers explain that they can't find what they are looking for. Also they don't understand the capabilities of the CLI so we need better ways to reveal that information.

* E.g., Zowe zos-files -h --examples
* The merge is sequential, depth first. That is for an action page, list out all the examples for the sub-objects. 
* Maybe include list of actions and sub objects at top as a summary or TOC
* Related to zowe --ac

**Comment by zFernand0** [comment]
This issue could be addressed by using `--help-web` in the future.
In the meantime, we can investigate ways of influencing the behavior of options in general.

**Comment by jelaplan** [comment]
I spoke with Stanko about it and we looked at showing examples at the action level of the hierarchy rather than the group. This scales down the number of options to a reasonable scannable amount. 

We also discussed the possibility of showing a single example from each leaf which I think is unnecessary at eh action level. But, it would be good at the group level to reduce the number of examples to a manageable number. 

I concur that this kind of capability in the Web Help would be good because then people could search. Rose, Mike, and I also discussed the possibility in that case to back into the detailed help pages from the example thus reversing the regular path from command --> examples  to  examples --> commands.
